model/Core/Classes/SoftwareAgent.md

```diff
@@ -0,0 +1,20 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# SoftwareAgent
+
+## Summary
+
+A software agent.
```

Comment:
```suggestion
A software representation of a user.
```
I think this gives a bit more information in the summary than just restating the class name.

Reply:
A software program given authority to perform an operation.

Reply:
Agent is "anything that has the potential to act.. This could be a person, organization, software agent, etc.
Agent is not limited to acting on a system but rather acting in any way or context."

So software agent has the potential to act, and the reason it has that potential is that it has authority, either by a system or anything else that recognizes that authority.